{{cookiecutter.project_id}}/terraform/stages/gke/main.tf

```diff
@@ -50,5 +50,5 @@ module "ingress" {
 
   # API domain, excluding protocols. E.g. example.com.
   api_domain        = var.api_domain
-  cors_allow_origin = "http://localhost:4200,http://localhost:3000,http://${var.web_app_domain},https://${var.web_app_domain}"
```

Comment:
this is where I suspect it will breaks. both localhost:4200 and :3000 are here intentionally. But since the build_template.sh uses "localhost" to replace all API_DOMAIN, and it incorrectly replace these too.

Let's merge this first and I will have next PR to fix it.